plugins/orchestrator-backend/src/service/DataIndexSwfService.ts

```diff
@@ -0,0 +1,81 @@
+import { cacheExchange, Client, fetchExchange } from '@urql/core';
+
+import { ErrorBuilder } from '../helpers/errorBuilder';
+import { BackendExecCtx } from '../types/backendExecCtx';
+import { DEFAULT_DATA_INDEX_URL } from '../types/constants';
+import { SonataWorkFlow } from '../types/SonataWorkFlow';
+
+export class DataIndexSwfService {
```

Comment:
yes, I'd say the same, to call swf or serverless-workflow to just workflow (more generic) and to avoid the term more specific like sonataflow or swf.
In this case I believe we can just call `DataIndexService`